packages/flutter/lib/src/widgets/_window_positioner.dart

```diff
@@ -0,0 +1,546 @@
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
+import '_window.dart';
+
+/// Defines how a child window will be placed relative to the anchor rectangle
+/// of its parent.
+///
+/// The specified anchor is used to derive an anchor point on the anchor rectangle that
+/// the child [BaseWindowController] will be positioned relative to.
+/// If a corner anchor is set (e.g. [topLeft] or [bottomRight]),
+/// the anchor point will be at the specified corner; otherwise, the derived anchor point
+/// will be centered on the specified edge, or in the center of the anchor rectangle
+/// if no edge is specified.
+///
+/// {@macro flutter.widgets.windowing.experimental}
+@internal
+enum WindowPositionerAnchor {
```

Comment:
I wonder if we should use [`Alignment`](https://api.flutter.dev/flutter/painting/Alignment-class.html) instead since its so similar?

One huge caveat: even though most people use `Alignment` like an `enum`, you can also do stuff like `Alignment(0.3, 0.7)`. But I suspect that might be OK?

Reply:
Thanks for the detailed thoughts. Using separate enums sounds good to me, closing!